index.js

```diff
@@ -99,6 +99,11 @@ export async function setupRuby(options = {}) {
   if (inputs['bundler-cache'] === 'true') {
     await common.time('bundle install', async () =>
       bundler.bundleInstall(gemfile, lockFile, platform, engine, version, bundlerVersion, inputs['cache-version']))
+
+    if (fs.existsSync(lockFile)) {
```

Comment:
```suggestion
    if (lockFile !== null && fs.existsSync(lockFile)) {
```
It feels a bit weird to call `fs.existsSync(null)` so let's be more explicit